src/components/Dashboard.vue

```diff
@@ -135,8 +139,12 @@ export default {
         });
     },
 
-    isMobileEventHandler(e) {
-      this.isMobile = window.innerWidth <= 1347;
+    resizeEventHandler(e) {
+      this.isMobile = window.innerWidth <= 440;
+      this.isTablet = window.innerWidth <= 878;
```

Comment:
Should this be the 976 number?

Reply:
Here we would want it to be 878 and not 976 because at the 878px breakpoint, the requirements bar would be hidden by default and toggleable which is what this.isTablet controls